tests/operators/test_sql_decorator.py

```diff
@@ -0,0 +1,60 @@
+import pathlib
+
+import pytest
+
+from astro.settings import SCHEMA
+from astro.sql.operators.sql_decorator import SqlDecoratedOperator
+from tests.operators import utils as test_utils
+
+CWD = pathlib.Path(__file__).parent
+
+
+@pytest.mark.parametrize(
+    "sql_server",
+    [
+        "snowflake",
+        "postgres",
+        "bigquery",
+        "sqlite",
+    ],
+    indirect=True,
+)
+@pytest.mark.parametrize(
+    "test_table",
+    [
+        {
+            "path": str(CWD) + "/../data/homes2.csv",
+            "load_table": True,
+            "is_temp": False,
+            "param": {
+                "schema": SCHEMA,
+                "table_name": test_utils.get_table_name("test"),
+            },
+        }
+    ],
+    indirect=True,
+)
+def test_sql_decorator_basic_functionality(sample_dag, sql_server, test_table):
+    """Test basic sql execution of SqlDecoratedOperator."""
+
+    def handler_func(result):
```

Comment:
I have a feeling this method may cease to exist in the refactoring, could we move this check to an assert, by the end of the test?